webtau-feature-testing/pom.xml

```diff
@@ -136,6 +136,7 @@
                 <artifactId>spring-boot-maven-plugin</artifactId>
                 <configuration>
                     <mainClass>com.example.demo.springboot.app.SpringBootDemoApp</mainClass>
+                    <wait>40000</wait>
```

Comment:
The default of 30s was not enough in my local development. Happy to revert this change if it works for everyone else.